RELEASE.md

```diff
@@ -0,0 +1,56 @@
+# How to make a release
+
+`oauthenticator` is a package [available on
+PyPI](https://pypi.org/project/oauthenticator/) and conda-forge. These are instructions on how
+to make a release on PyPI.
+
+For you to follow along according to these instructions, you need:
+- To be a maintainer of the [PyPI oauthenticator
+  project](https://pypi.org/project/oauthenticator/).
+- To have push rights to the [oauthenticator GitHub
+  repository](https://github.com/jupyterhub/oauthenticator).
+
+## Technical steps to make a release
+
+1. Checkout master and make sure it is up to date.
+
+   ```
+   git checkout master
+   git fetch <upstream> master
+   git reset --hard <upstream>/master
+   ```
+
+1. Update [CHANGELOG.md](CHANGELOG.md). Doing this can be made easier with the
+   help of the
+   [choldgraf/github-activity](https://github.com/choldgraf/github-activity)
+   utility.
+
+1. Set the `version_info` variable in [_version.py](oauthenticator/_version.py)
+   appropriately and make a commit with message `release <tag>`.
+
+1. Create a git tag for the commit.
+
+   ```
+   git tag -a $TAG -m $TAG
+   ```
+
+1. Package the release
+   ```
+   python3 setup.py sdist bdist_wheel
```

Comment:
I followed the setup from travis:
https://github.com/jupyterhub/oauthenticator/blob/0d2b44c2b7f08090ac5cb07c7577c02abdece480/.travis.yml#L11-L13

```$ python3 setup.py sdist bdist_wheel
/oauthenticator-venv/lib/python3.7/site-packages/setuptools/dist.py:475: UserWarning: Normalizing '0.10.0dev' to '0.10.0.dev0'
  normalized_version,
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: invalid command 'bdist_wheel'
```
Probably `pip install wheel` needs to be added to the instructions or a requirements file?